app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointment.kt

```diff
@@ -62,5 +66,53 @@ data class OverdueAppointment(
           ORDER BY A.scheduledDate, A.updatedAt ASC
           """)
     fun appointmentsForFacility(facilityUuid: UUID, scheduledStatus: Appointment.Status, dateNow: LocalDate): Flowable<List<OverdueAppointment>>
+
+    /**
+     * FYI: IntelliJ's SQL parser highlights `isAtHighRisk` as an error, but it's not. This is probably
```

Comment:
Ugh, I hate this. Even though we know that whatever we are doing is fine, the query parser doesn't know this since it doesn't know we are using SQLite under the hood.

Reply:
I don't think it has anything to do with Android. Room uses IntelliJ's parser so technically it shouldn't target any specific version of Sqlite.